qframe.go

```diff
@@ -1022,6 +1039,37 @@ func (qf QFrame) ToJSON(writer io.Writer) error {
 	return err
 }
 
+// ToSQL writes a QFrame into a SQL database.
+func (qf QFrame) ToSQL(tx *sql.Tx, table string) error {
+	if qf.Err != nil {
+		return errors.Propagate("ToSQL", qf.Err)
+	}
+	columns := qf.ColumnNames()
+	builders := make([]qfsqlio.ArgBuilder, len(columns))
+	for i, name := range columns {
+		builder, err := qfsqlio.NewArgBuilder(qf.columnsByName[name].Column)
```

Comment:
Nit pick: The assignment could probably be `builders[i], err = ...` to save a line.